controllers/airflowcluster_controller.go

```diff
@@ -615,6 +615,8 @@ func gitContainer(s *alpha1.GitSpec, volName string) (bool, corev1.Container) {
 		{Name: "GIT_SYNC_BRANCH", Value: s.Branch},
 		{Name: "GIT_SYNC_ONE_TIME", Value: strconv.FormatBool(s.Once)},
 		{Name: "GIT_SYNC_REV", Value: s.Rev},
+		{Name: "GIT_SYNC_SUBMODULES", Value: s.SubmoduleMode},
```

Comment:
Can you wrap this in a conditional so that it is only passed when it exists in the `AirflowCluster` custom resource.

Reply:
@LaVLaS this is also done